### PR TITLE
Release 1.3.0: Compatibility with zserio>=2.6.0

### PR DESCRIPTION
**Pull request by josephbirkner**
### Changes

The zserio runtime API changed a bit in 2.6.0, so zswag needs to adapt.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,5 +2,5 @@ connexion
 requests
 zserio>=2.4.2
```
Comment (MisterGC) [inline requirements.txt:3]: The C++ impl. relies on zserio 2.5+ runtime - is there any reason why `zserio>=2.4.2` should be kept here?

Comment (josephbirkner) [reply requirements.txt:3]: The Python server/client are not so sensitive to the zserio version, anything between zserio 2.4.2 to zserio 2.7.0 (latest) works (verified).

**Review by MisterGC — APPROVED** [review]

**Comment by fklebert** [comment]
Ist that PR also valid for zswag 2.7.0 
@MisterGC have you tested this?

**Comment by josephbirkner** [comment]
> Ist that PR also valid for zswag 2.7.0 @MisterGC have you tested this?

Now it is 😄

**Comment by fklebert** [comment]
> > Ist that PR also valid for zswag 2.7.0 @MisterGC have you tested this?
> 
> Now it is 😄

Awesome! 👍

**Review by fklebert — APPROVED** [review]

**Pull request merged**